### PR TITLE
Adjust code and test since HTTP1 capitalization has been fixed

### PR DESCRIPTION
**Pull request by carloseltuerto**
HTTP1 capitalization has been fixed, so all the ```toLowerCase``` were removed.
The "number of byte received" check as been disabled in order to enable encompassing tests

Description: Adjust code and test since HTTP1 capitalization has been fixed
Risk Level: None
Testing: yes
Docs Changes: N/A
Release Notes: N/A
Signed-off-by: Charles Le Borgne cleborgne@google.com

### CODE REVIEW

File: library/java/org/chromium/net/impl/CronetUrlRequest.java [hunk 1]
```diff
@@ -403,11 +403,11 @@ private void onResponseHeaders(ResponseHeaders responseHeaders, boolean lastCall
       return;
     }
     final List<Map.Entry<String, String>> headerList = new ArrayList<>();
-    String selectedTransport = "http/1.1"; // TODO(carloseltuerto) looks dubious
```
Comment (alyssawilk) [inline library/java/org/chromium/net/impl/CronetUrlRequest.java:406]: Is this the ALPN issue? Should we leave it with a TODO?

**Review by alyssawilk — APPROVED** [review]
Yay progress!

File: library/java/org/chromium/net/impl/CronetUrlRequest.java (hunk 1 above)
Comment (carloseltuerto) [reply library/java/org/chromium/net/impl/CronetUrlRequest.java:406]: I added a todo further down, line 431

``` // TODO(https://github.com/envoyproxy/envoy-mobile/issues/1546) negotiated protocol```

For HTTP1, the logic here is fine: the expected value is ```"unknown"``` - that's what the original test is looking for.

For HTTP2/3, I don't have tests yet, as I need to make TLS to work for tests.

**Review by buildbreaker — APPROVED** [review]
🎉

**Pull request merged**